docker/build/cicd.centos8.Dockerfile

```diff
@@ -0,0 +1,35 @@
+ARG ARCH="amd64"
+
+FROM ${ARCH}/centos:8
+ARG GOLANG_VERSION
+ARG ARCH="amd64"
+RUN yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm && \
```

Comment:
Why do we `yum install` this but `dnf install` later? Shouldn't it all be dnf?

Reply:
it shouldn't actually make a difference, but it wouldn't hurt to switch to dnf

Reply:
Replaced use of yum with dnf.

Reply:
updated to use dnf.